Sources/MapboxMaps/Style/Style+Localization.swift

```diff
@@ -62,6 +86,10 @@ extension Style {
             }
         }
 
+        if !supportedLocaleIdentifiers.contains(locale.identifier) {
+            return locale.languageCode!
+        }
+
         return locale.identifier
```

Comment:
I think this would be more clear if you negate the logic and write it with a ternary operator:

```suggestion
    return supportedLocaleIdentifiers.contains(locale.identifier) ? locale.identifier : locale.languageCode!
```